tests/unit/config/test_logging_config.py

```
@@ -0,0 +1,211 @@
+    """
+    Ensure logging state is clean before and after each test to avoid cross-test pollution.
+    Removes and closes handlers from root and 'snoop' logger and resets levels.
+    """
+    # Before test
+    def _cleanup():
+        # Root handlers
+        for h in list(logging.root.handlers):
+            try:
+                logging.root.removeHandler(h)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)